src/spec/spec_helper.rb

```diff
@@ -1,13 +1,13 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'test'
 
+require 'simplecov'
 if ENV['CI']
-  require 'simplecov'
   require 'coveralls'
   SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-  SimpleCov.start :rails do
-    add_filter "/spec/"
-  end
+end
+SimpleCov.start :rails do
```

Comment:
@jhsu802701 Does this mean that SimpleCov runs on every spec run? I'd prefer to either keep it only running under CI or if explicitly called running locally. Thoughts?

Reply:
I'd like to have SimpleCov run locally by default.  It doesn't seem to significantly add to the time required to run the test suite, and I cannot think of a good reason it should be disabled by default.

Reply:
Alright. That works for me.